### ISSUE TITLE
Consider adding chronicled wishes after it is more understood

### ISSUE THREAD

**Issue by benlei** [benlei/samsara-web#141]
https://genshin-impact.fandom.com/wiki/Category:Chronicled_Wishes

The [first](https://genshin-impact.fandom.com/wiki/Ode_to_the_Dawn_Breeze/2024-03-13) chronicled wish seems to only last as long as the first banner... is this going to be the same every patch? Will there be a 2nd half?

At the moment, probably best to just show 5 star pool and nothing else?

**Comment by benlei** [comment]
For the first chronicled wish... seems like no 2nd part. Off top of head... and fandom wise it seems like the start/end date matches the first banner too. This may be too early to say, but perhaps I can just "merge" the limited 5 star chars + weaps and chronicled chars + weaps.

**Comment by benlei** [comment]
https://genshin-impact.fandom.com/api.php?action=query&generator=categorymembers&gcmtitle=Category:Chronicled_Wishes&prop=categories&cllimit=max&gcmlimit=max&format=json

**Comment by benlei** [comment]
```json
{
  "batchcomplete": "",
  "limits": {
    "categorymembers": 500,
    "categories": 500
  },
  "query": {
    "pages": {
      "295596": {
        "pageid": 295596,
        "ns": 0,
        "title": "Ode to the Dawn Breeze/2024-03-13",
        "categories": [
          {
            "ns": 14,
            "title": "Category:Chronicled Wishes"
          },
          {
            "ns": 14,
            "title": "Category:Event Wishes"
          },
          {
            "ns": 14,
            "title": "Category:Ode to the Dawn Breeze"
          },
          {
            "ns": 14,
            "title": "Category:Past Events"
          },
          {
            "ns": 14,
            "title": "Category:Released in Version 4.5"
          },
          {
            "ns": 14,
            "title": "Category:Removed"
          },
          {
            "ns": 14,
            "title": "Category:Version"
          },
          {
            "ns": 14,
            "title": "Category:Wish"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Albedo"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Alley Hunter"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Amber"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Aquila Favonia"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Barbara"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Beacon of the Reed Sea"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Bennett"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Black Tassel"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Bloodtainted Greatsword"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Cool Steel"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Debate Club"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Diluc"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Diona"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Dragon's Bane"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Emerald Orb"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Eula"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Eye of Perception"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Favonius Codex"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Favonius Greatsword"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Favonius Lance"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Favonius Sword"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Favonius Warbow"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Ferrous Shadow"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Fischl"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Harbinger of Dawn"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Hunter's Path"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Jean"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Kaeya"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Klee"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Lion's Roar"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Lisa"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Lost Prayer to the Sacred Winds"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Magic Guide"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Mika"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Mitternachts Waltz"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Mona"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Noelle"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Rainslasher"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Raven Bow"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Razor"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Rosaria"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Rust"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Sacrificial Bow"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Sacrificial Fragments"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Sacrificial Greatsword"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Sacrificial Sword"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Sharpshooter's Oath"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Skyrider Sword"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Skyward Atlas"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Skyward Blade"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Skyward Harp"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Skyward Pride"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Skyward Spine"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Slingshot"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Song of Broken Pines"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Sucrose"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes The Alley Flash"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes The Bell"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes The Flute"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes The Stringless"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes The Widsith"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Thrilling Tales of Dragon Slayers"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Wine and Song"
          },
          {
            "ns": 14,
            "title": "Category:Wish Pool Includes Wolf's Gravestone"
          },
          {
            "ns": 14,
            "title": "Category:Wishes"
          }
        ]
      },
      "295597": {
        "pageid": 295597,
        "ns": 0,
        "title": "Chronicled Wish",
        "categories": [
          {
            "ns": 14,
            "title": "Category:Article Stubs"
          },
          {
            "ns": 14,
            "title": "Category:Chronicled Wishes"
          },
          {
            "ns": 14,
            "title": "Category:DE Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:KO Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:Pages Using DPL Parser Function"
          },
          {
            "ns": 14,
            "title": "Category:Released in Version 4.5"
          },
          {
            "ns": 14,
            "title": "Category:TH Romanization Missing"
          },
          {
            "ns": 14,
            "title": "Category:TH Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:Terminology"
          },
          {
            "ns": 14,
            "title": "Category:VI Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:Version"
          },
          {
            "ns": 14,
            "title": "Category:Wish Types"
          }
        ]
      },
      "295710": {
        "pageid": 295710,
        "ns": 0,
        "title": "Ode to the Dawn Breeze",
        "categories": [
          {
            "ns": 14,
            "title": "Category:Chronicled Wishes"
          },
          {
            "ns": 14,
            "title": "Category:DE Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:KO Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:Pages Using DPL Parser Function"
          },
          {
            "ns": 14,
            "title": "Category:TH Romanization Missing"
          },
          {
            "ns": 14,
            "title": "Category:TH Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:TR Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:VI Translation Missing"
          },
          {
            "ns": 14,
            "title": "Category:Wish Series"
          }
        ]
      }
    }
  }
}
```

**Comment by benlei** [comment]
completed
